sickbeard/helpers.py

```diff
@@ -114,65 +114,81 @@ def remove_non_release_groups(name, clean_proper=False):
     # Check your database for funky release_names and add them here, to improve failed handling, archiving, and history.
     # select release_name from tv_episodes WHERE LENGTH(release_name);
     # [eSc], [SSG], [GWC] are valid release groups for non-anime
+
+    # All lowercase and alphabetically sorted
+    groups = {
+        '-20-40',
+        '1044',
+        'agusiq',
+        'androidtwou',
```

Comment:
Yesterday I looked at some real cases and also tried this approach. There are some gotchas like non_group names with delimiter characters  (e.g.: `. `) that can make it much harder. 

I would like to take a look on a good way to handle this during my free time. But I would also like to have a lot o real examples in order to have them covered by tests. Could you guys provide me your cache.db file so I can have a big data set. I'll be on irc later today